src/Features/CSharp/Portable/UpgradeProject/CSharpUpgradeProjectCodeFixProvider.cs

```diff
@@ -39,16 +38,7 @@ internal class CSharpUpgradeProjectCodeFixProvider : AbstractUpgradeProjectCodeF
         public override ImmutableArray<string> SuggestedVersions(ImmutableArray<Diagnostic> diagnostics)
```

Comment:
seems like this should just be SuggestedVersion and return just a string/null.

Reply:
If that happens, it looks like a whole lot of code in the base class gets simplified.

Reply:
💭 I'm not convinced we won't want to bring back some form of "multiple options" in the future.